plugin/al/lib/AL_USDMaya/AL/usdmaya/nodes/ProxyShape.cpp

```diff
@@ -1449,6 +1451,8 @@ void ProxyShape::loadStage()
         }
     }
 
+    MayaUsd::ufe::refreshStages();
```

Comment:
The Autodesk plugin also loads stages, and doesn't explicitly force the revoking of USD listeners and rebuilding of the stage map.  What is different in the AL plugin?